dotenv.py

```diff
@@ -86,7 +86,12 @@ def parse_dotenv(dotenv_path):
             if not line or line.startswith('#') or '=' not in line:
                 continue
             k, v = line.split('=', 1)
-            v = v.strip("'").strip('"')
+            v = str(v).replace("'", "").replace('"', '').replace(" ", "")
+            if v.lower() == 'true':
+                v = True
+            elif v.lower() == 'false':
+                v = False
```

Comment:
We can't support type conversion as the goal of the this library to only populate the environment with variables in the same way it might have been added by any other method. So any other tool reading the `os.environ` always works with the expectation that it's going to be string. 

TLDR; Type conversions must be explicit. And it should be responsibility of the tool reading the values.